helios-client/src/main/java/com/spotify/helios/common/descriptors/PortMapping.java

```diff
@@ -41,33 +44,54 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class PortMapping extends Descriptor {
 
+  public static final String WILDCARD_ADDRESS = "0.0.0.0";
   public static final String TCP = "tcp";
   public static final String UDP = "udp";
 
+  // Nullable for backwards compatibility
+  // TODO (dxia) Should we validate this string?
```

Comment:
I think this could be useful - if so you can use `boolean com.google.common.net.InetAddresses.isInetAddress(String)`, which avoids any DNS lookups